draft-ietf-dnsop-dns-capture-format-00.md

```diff
@@ -1365,8 +1362,8 @@ where each query/response pair is represented in a single record
 entry, and a block structure as described above, where query/responses
```

Comment:
s/a block structure as described above/a block structure based on the C-DNS format/